### PR TITLE
Path: Added KineticNC Postprocessor

### PR DESCRIPTION
**Pull request by SamuelMayer**
I was asked to create a pull request for the KineticNC Postprocessor here: https://forum.freecadweb.org/viewtopic.php?f=15&t=57003&p=568889#p568889

So I did :)

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [ ]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [ ]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [ ]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [ ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [ ]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [ ]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the [FreeCAD bug tracker issue number](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) in case a particular commit solves or is related to an existing issue on the tracker. Ex: `Draft: fix typos - fixes #0004805`

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.20 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=56135).

---

### CODE REVIEW

File: src/Mod/Path/PathScripts/post/KineticNC_post.py [hunk 1]
```diff
@@ -0,0 +1,365 @@
+# ***************************************************************************
+# *   (c) sliptonic (shopinthewoods@gmail.com) 2014                        *
```
Comment (luzpaz) [inline src/Mod/Path/PathScripts/post/KineticNC_post.py:2]: Should be:  
`# *  Copyright (c) 2014 sliptonic <shopinthewoods@gmail.com>                         *`

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5508">pipeline status</a> for feature branch [PR_5508](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5508). Pipeline [464825522](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/464825522) was triggered at [fa5dc04](https://github.com/FreeCAD/FreeCAD/pull/5508/commits/fa5dc04d2a83c8c270b1240b6570adfcf256fd84). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/Path/PathScripts/post/KineticNC_post.py (hunk 1 above)
Comment (SamuelMayer) [reply src/Mod/Path/PathScripts/post/KineticNC_post.py:2]: Do I have to make a new pull request with these changes? Sorry if that is a stupid question, this is my first pull request ever

Comment (luzpaz) [reply src/Mod/Path/PathScripts/post/KineticNC_post.py:2]: No unnecessary, in the same branch that you've submitted, made the appropriate edit. Then type:
```
git add .
git commit -m "Added requested revision"
git push
```

But you've made the mistake using the master branch to submit your changes and not making a new branch. You probably don't have a Github token setup either to  `git push` from the command line. So whomever the dev that merges this should make the changes for you.

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5508">pipeline status</a> for feature branch [PR_5508](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_5508). Pipeline [465402050](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/465402050) was triggered at [3d62aeb](https://github.com/FreeCAD/FreeCAD/pull/5508/commits/3d62aeba37c628eefa8b08f2662db5cc10179e11). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/Path/PathScripts/post/KineticNC_post.py (hunk 1 above)
Comment (sliptonic) [reply src/Mod/Path/PathScripts/post/KineticNC_post.py:2]: You can also make the changes locally and amend the existing commit.  Then do a git push --force 
Github still detects the change and will update the PR

Comment (SamuelMayer) [reply src/Mod/Path/PathScripts/post/KineticNC_post.py:2]: At this point it seems the easiest to create a proper pull request from the command line and not via file upload. I will close this request.

**Pull request closed without merging**